lbry/schema/mime_types.py

```diff
@@ -162,6 +163,7 @@
 
 
 def guess_media_type(path):
+    # should we be using "file --mime-type -b $filename" on linux?
```

Comment:
Can you ask this in the PR discussion or in a new issue where it can be answered / discussed?

Reply:
I've since realized that there is a built-in mimetypes module that we should probably use instead.

Reply:
yeah, that's how it worked originally but it was missing types and so we made a our own custom lookup